scalalib/test/resources/hello-world/core/src/Main.scala

```diff
@@ -10,7 +13,9 @@ object Main0{
   }
 }
 object Main extends App {
-
+  new JButton("hello from javax")
```

Comment:
Could we have a comment here saying what this is testing for? And perhaps a link back to issue 251